### PR TITLE
persist,storage: embellish tracing/spans

### PR DESCRIPTION
**Pull request by aljoscha**

### CODE REVIEW

File: src/dataflow-types/src/client/controller/storage.rs [hunk 1]
```diff
@@ -493,6 +493,7 @@ where
         Ok(())
     }
 
+    #[tracing::instrument(level = "debug", skip(self))]
```
Comment (danhhz) [inline src/dataflow-types/src/client/controller/storage.rs:496]: I think all these want to be skip_all, the fn args would be pretty expensive to serialize

File: src/persist-client/src/impl/machine.rs [hunk 2]
```diff
@@ -316,7 +316,7 @@ where
                             .truncate(Instant::now() + FOREVER, &path, self.state.seqno())
                             .await
                     })
-                    .instrument(trace_span!("apply_unbatched_cmd::truncate"))
+                    .instrument(debug_span!("apply_unbatched_cmd::truncate"))
```
Comment (danhhz) [inline src/persist-client/src/impl/machine.rs:319]: I might yolo this and a couple other tweaks in myself today :)

**Review by danhhz — APPROVED** [review]
LGTM modulo the skip_all comment

File: src/persist-client/src/impl/machine.rs (hunk 2 above)
Comment (danhhz) [reply src/persist-client/src/impl/machine.rs:319]: nevermind, I just added them to this PR XD

Comment (aljoscha) [reply src/persist-client/src/impl/machine.rs:319]: :bow:

**Pull request merged**